<?xml version="1.0"?>
<launch>
  <include file="$(find force_local_planner)/launch/navigation_map.launch"/>

  <include file="$(find force_local_planner)/launch/fake_localization.launch"/>

	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
		<rosparam file="$(find force_local_planner)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find force_local_planner)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find force_local_planner)/params/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find force_local_planner)/params/global_costmap_params.yaml" command="load" />
		<rosparam file="$(find force_local_planner)/params/move_base_params.yaml" command="load" />
		<rosparam file="$(find force_local_planner)/params/force_local_planner_params.yaml" command="load" />

    <remap from="odom" to="/mobile_base_controller/odom"/>
    <remap from="map" to="/map"/>
    <remap from="cmd_vel" to="/mobile_base_controller/cmd_vel"/>
		
		<param name="base_global_planner" type="string" value="navfn/NavfnROS" />
		<param name="base_local_planner" value="force_local_planner/ForceLocalPlanner" />
	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find force_local_planner)/rviz/rviz.rviz" />
</launch>